Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml

```diff
@@ -110,6 +110,7 @@
   - type: Tag
     tags:
       - FoodSnack
+      - ReptilianFood
```

Comment:
I gave it another in-game test and what this tag does is making it possible for Lizards to eat certain items, it has nothing to do with metabolism. 
If you want them to be able to eat chocolate bars you would have to add it to the prototype below for the unwrapped bar. This one is for the wrapped chocolate bar.
```suggestion
```

Reply:
Fixing that